package.json

```diff
@@ -2,19 +2,17 @@
   "name": "propsd",
   "version": "3.0.3",
   "description": "A local daemon for loading static and dynamic properties",
-  "main": "lib/index.js",
-  "directories": {
-    "doc": "docs",
-    "bin": "bin"
-  },
   "scripts": {
-    "start": "./bin/server.js",
-    "test": "mocha",
-    "lint": "eslint bin/** lib/** test/**",
-    "coverage": "istanbul cover _mocha -- -R spec",
+    "start": "nodemon ./src/bin/server.js -r babel-require",
+    "clean": "rimraf dist",
+    "version": "./test/bin/version.js",
+    "transpile": "npm run version && babel src --out-dir dist --copy-files",
+    "test": "npm run transpile && mocha",
+    "lint": "eslint src/**/*.js test/**/*.js",
+    "coverage": "npm run transpile && istanbul cover _mocha -- -R spec",
     "report-coverage": "npm run coverage && cat ./coverage/lcov.info | coveralls && rm -rf ./coverage",
-    "metadata-server": "./test/bin/metadata-server",
-    "s3-server": "./test/bin/s3-server.js",
+    "metadata-server": "nodemon ./test/bin/metadata-server",
```

Comment:
Can we change the command to `nodemon ./test/bin/metadata-server --` so arguments passed on the command line to `npm run metadata-server` are passed to the metadata-server instead of nodemon?